src/mame/drivers/viper.cpp

```diff
@@ -130,19 +130,19 @@ Games on this hardware include:
 
 Konami
 Game ID  Year    Game
--------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------------------------------------------------
 GK922    2000    Code One Dispatch
 G????    2001    ParaParaParadise 2nd Mix
-GM941    2001    GTI Club 2
-G?A00    2001    Police 911 (USA) / Police 24/7 (World) / Keisatsukan Shinjuku 24ji (Japan)
+GM941    2000    Driving Party: Racing in Italy (World) / GTI Club: Corso Italiano (Japan) / GTI Club 2 (USA?)
+G?A00    2000    Police 911 (USA) / Police 24/7 (World) / The Keisatsukan: Shinjuku 24-ji (Asia/Japan/Korea)
```

Comment:
What evidence do you have that the Japanese title is used for Korea?  Remember Korea had very strict laws banning various forms of “Japanese cultural imports”, including the use of Japanese text in entertainment media.

Reply:
I have added a couple of YouTube videos in one of my previous comments for this recently to show proof.

Here it is again.
https://www.youtube.com/watch?v=MFRB21gcHxk

Reply:
Sorry, I didn’t know at first but looked it up later.  So true, Korea is nit picky with specific Japanese media.

Reply:
Sorry, I didn’t know at first but looked it up later. So true, Korea does find certain Japanese media controversial.

Reply:
Also made a MAME snapshot for additional evidence that the Japanese title on Korean sets exist.
![0003](https://user-images.githubusercontent.com/95501796/152666073-b016e9ed-aa3a-4051-91c6-2cf869e5c8de.png)

Reply:
I wouldn’t necessarily trust a MAME screenshot though – bad dump and emulation issues are a thing.

Reply:
Did you check out the video?

Reply:
Oh, wait more evidence from other video links.
https://youtu.be/Hfqnm8fLDr4
https://youtu.be/r5UedHQ5brQ

Reply:
One more for evidence via video link about the Korean sets with the Japanese title.
https://youtu.be/BmYGJXHYObs

Reply:
Here’s a mention also about The Keisatsukan name in the Korean sets.
https://www.gtplanet.net/forum/threads/old-school-the-keisastsukan-2000-arcade.68705/

Reply:
So yep.  Brought these up as evidence that The Keisatsukan title is the most correct name for the Korean release.

Reply:
So yep. Brought these links from earlier up as evidence that The Keisatsukan title is the most correct name for the Korean releases (p911k and p911kc).